hnn_core/tests/test_compare_hnn.py

```diff
@@ -43,7 +43,7 @@ def run_hnn_core(backend=None, n_jobs=1):
     net_reduced = Network(params_reduced)
 
     if backend == 'mpi':
-        with MPIBackend(n_procs=2, mpi_cmd='mpiexec'):
+        with MPIBackend(mpi_cmd='mpiexec'):
```

Comment:
should we rename this file from test_compare_hnn to test_parallel_backends so we have a one to one mapping between files and their tests?